gremlin-go/driver/driverRemoteConnection.go

```diff
@@ -87,7 +87,7 @@ func NewDriverRemoteConnection(
 
 	logHandler := newLogHandler(settings.Logger, settings.LogVerbosity, settings.Language)
 	if settings.Session != "" {
-		logHandler.log(Info, sessionDetected)
+		logHandler.log(Debug, sessionDetected)
```

Comment:
The Client seems to have the same printout, do we want that?

Reply:
The section from 89-92 is replicated in `client.go`. `NewDRC()` doesn't depend on `NewClient()` and operates independently as it generates its own Client object directly. We could utilize the `NewClient` function in `NewDRC`, but I'm wondering if there was a reason it was intentionally made that way.